aspnetcore/toc.md

```diff
@@ -33,67 +32,89 @@
+#### [Examining the Details and Delete methods](tutorials/first-mvc-app/details.md)
+
+### [Create a web app on a Mac](tutorials/first-mvc-app/index.md)
+#### [Getting started](tutorials/first-mvc-app-mac/start-mvc.md)
+#### [Adding a controller](tutorials/first-mvc-app-mac/adding-controller.md)
+#### [Adding a view](tutorials/first-mvc-app-mac/adding-view.md)
+#### [Adding a model](tutorials/first-mvc-app-mac/adding-model.md)
+#### [Working with SQL Server LocalDB](tutorials/first-mvc-app-mac/working-with-sql.md)
+#### [Controller methods and views](tutorials/first-mvc-app-mac/controller-methods-views.md)
+#### [Adding Search](tutorials/first-mvc-app-mac/search.md)
+#### [Adding a New Field](tutorials/first-mvc-app-mac/new-field.md)
+#### [Adding Validation](tutorials/first-mvc-app-mac/validation.md)
+#### [Examining the Details and Delete methods](tutorials/first-mvc-app/details.md)
+
+### [Create a web app with VS Code](tutorials/first-mvc-app-xplat/index.md)
+#### [Getting started](tutorials/first-mvc-app-xplat/start-mvc.md)
+#### [Adding a controller](tutorials/first-mvc-app-xplat/adding-controller.md)
+#### [Adding a view](tutorials/first-mvc-app-xplat/adding-view.md)
+#### [Adding a model](tutorials/first-mvc-app-xplat/adding-model.md)
+#### [Working with SQL Server LocalDB](tutorials/first-mvc-app-xplat/working-with-sql.md)
+#### [Controller methods and views](tutorials/first-mvc-app-xplat/controller-methods-views.md)
+#### [Adding Search](tutorials/first-mvc-app-xplat/search.md)
+#### [Adding a New Field](tutorials/first-mvc-app-xplat/new-field.md)
+#### [Adding Validation](tutorials/first-mvc-app-xplat/validation.md)
+#### [Examining the Details and Delete methods](tutorials/first-mvc-app/details.md)
 
+### [Create a Web API](xref:tutorials/first-web-api-mac)
+### [Data access - working with EF Core](data/ef-mvc/index.md)
+#### [Getting started](data/ef-mvc/intro.md)
+#### [Create, Read, Update, and Delete operations](data/ef-mvc/crud.md)
+#### [Sorting, filtering, paging, and grouping](data/ef-mvc/sort-filter-page.md)
+#### [Migrations](data/ef-mvc/migrations.md)
+#### [Creating a complex data model](data/ef-mvc/complex-data-model.md)
+#### [Reading related data](data/ef-mvc/read-related-data.md)
+#### [Updating related data](data/ef-mvc/update-related-data.md)
+#### [Handling concurrency conflicts](data/ef-mvc/concurrency.md)
+#### [Inheritance](data/ef-mvc/inheritance.md)
+#### [Advanced topics](data/ef-mvc/advanced.md)
+### [Creating backend services for mobile apps](mobile/native-mobile-backend.md)
+### [Building Web APIs](mvc/web-api/index.md)
+#### [Create a Web API with ASP.NET Core MVC using Visual Studio](tutorials/first-web-api.md)
+#### [ASP.NET Web API Help Pages using Swagger](tutorials/web-api-help-pages-using-swagger.md)
+#### [Creating backend services for native mobile applications](mobile/native-mobile-backend.md)
+#### [Formatting response data](mvc/models/formatting.md)
```

Comment:
Formatting response data and custom formatters don't belong under tutorials.  I'd move them to advanced